@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/index.js

```diff
@@ -23,10 +23,10 @@ const DownloadModalContent = ({ iiifManifest }) => {
           <div sx={sx.metadata}>
             <DownloadMetadata iiifManifest={iiifManifest} />
           </div>
-          {
-          // <Styled.h2 sx={sx.header}>Citation</Styled.h2>
-          // <DownloadCitation iiifManifest={iiifManifest} />
-          }
+          <Styled.h2 sx={sx.header}>Citation</Styled.h2>
```

Comment:
This line likely needs to get moved into the component so we're not showing a header when there is no content displayed. Don't forget to copy the sx style.

http://localhost:8000/item/004789783
![Screen Shot 2020-06-12 at 2 25 27 PM](https://user-images.githubusercontent.com/416559/84534807-9ad79100-acb8-11ea-8974-a51a4bdd86d7.png)